docs/developer-guide/events.md

```diff
@@ -0,0 +1,23 @@
+# Website events
+
+
+### event lists
+<table>
```

Comment:
Please use Markdown syntax for creating tables.
See https://raw.githubusercontent.com/spaship/operator/main/docs/operator-manual/configuration.md

Reply:
okay sure